src/editors/containers/LibraryContentEditor/data/urls.js

```diff
@@ -0,0 +1,19 @@
+export const v1Libraries = ({ studioEndpointUrl }) => (
+  `${studioEndpointUrl}/api/contentstore/v1/home`
+);
+
+export const v2Libraries = ({ studioEndpointUrl }) => (
+  `${studioEndpointUrl}/api/libraries/v2/`
+);
+
+export const libraryMetadata = ({ studioEndpointUrl, libraryId }) => (
```

Comment:
note: v1 library metadata will come from a different API.

Reply:
I'll rename these urls and request apis to be specific to library version